config/samples/kmm_v1beta1_managedclustermodule.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kmm.sigs.k8s.io/v1beta1
```

Comment:
What is that file for?

Reply:
This was scaffolded by `kubebuilder` when I created the CRD. It's created [in order to be used by developers](https://book.kubebuilder.io/cronjob-tutorial/running.html?highlight=samples#optional), when trying to test the operator.   

Is this helpful?

Reply:
Yup. Thanks again.